TrafficCapture/dockerSolution/src/main/docker/migrationConsole/lib/console_link/tests/test_cli.py

```diff
@@ -51,6 +51,35 @@ def test_cli_cluster_cat_indices(runner, env, mocker):
     mock.assert_called()
 
 
+def test_cli_cluster_connection_check(runner, env, mocker):
+    mock = mocker.patch('console_link.logic.clusters.connection_check')
+    result = runner.invoke(cli, ['--config-file', str(VALID_SERVICES_YAML), 'clusters', 'connection-check'],
+                           catch_exceptions=True)
+    # Should have been called two times.
+    assert result.exit_code == 0
+    assert 'SOURCE CLUSTER' in result.output
+    assert 'TARGET CLUSTER' in result.output
+    mock.assert_called()
+
+
+def test_cli_cluster_run_test_benchmarks(runner, env, mocker):
+    mock = mocker.patch('console_link.logic.clusters.run_test_benchmarks')
+    result = runner.invoke(cli, ['--config-file', str(VALID_SERVICES_YAML), 'clusters', 'run-test-benchmarks'],
+                           catch_exceptions=True)
+    mock.assert_called_once()
+    assert result.exit_code == 0
+
+
+def test_cli_cluster_clear_indices(runner, env, mocker):
```

Comment:
can you do a version of this test _without_ `--acknowledge-risk` to make sure that it doesn't call clear?

Reply:
Added 👍